2 Business layer/CandidateEvaluator.Core/Handlers/Queries/GetQuestionHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CandidateEvaluator.Contract.Handlers;
+using CandidateEvaluator.Contract.Models;
+using CandidateEvaluator.Contract.Queries.Question;
+using CandidateEvaluator.Contract.Repositories;
+using System.Threading.Tasks;
+
+namespace CandidateEvaluator.Core.Handlers.Queries
+{
+    public class GetQuestionHandler : IQueryHandler<GetQuestion, Question>
+    {
+        private readonly IQuestionRepository _modelRepository;
+
+        public GetQuestionHandler(IQuestionRepository modelRepository)
+        {
+            _modelRepository = modelRepository;
+        }
+
+        public async Task<Question> HandleAsync(GetQuestion query)
+        {
+            return await _modelRepository.Get(query.OwnerId, query.CategoryId, query.Id);
```

Comment:
If you are just returning output from a `Task`, there is no point to await that. Please use construction below:
```
        public Task<Question> HandleAsync(GetQuestion query)
        {
            return _modelRepository.Get(query.OwnerId, query.CategoryId, query.Id);
        }
```

Reply:
This is next reason, why we shouldn't use `Async` suffix. We won't use async everywhere we use Tasks.

Reply:
Done